src/components/appNavigation/TabsView.js

```diff
@@ -224,6 +224,7 @@ const TabsView = React.memo(({ navigation }) => {
         </>
       )}
       {/*{!showSupportFirst && <SupportButton onPress={goToSupport} style={supportButtonStyles} />}*/}
+      <Appbar.Content />
```

Comment:
@mauriguz what does this do?
does it still look good when other buttons are enabled?

Reply:
@mauriguz @MathiasMoralesDixtra

Reply:
@sirpy the side menu icon was at the left instead of right, this adds spacing and moves it to the left